File: unidbg-android/src/main/java/com/vortexdbg/linux/thread/AndroidWaiter.java
package com.vortexdbg.linux.thread;

import com.vortexdbg.Emulator;
import com.vortexdbg.thread.AbstractWaiter;
import com.vortexdbg.thread.Waiter;

public abstract class AndroidWaiter extends AbstractWaiter implements Waiter {

    @Override
    public void onContinueRun(Emulator<?> emulator) {
    }
}

File: vortexdbg-android/src/main/kotlin/com/vortexdbg/linux/thread/AndroidWaiter.kt
package com.vortexdbg.linux.thread

import com.vortexdbg.Emulator
import com.vortexdbg.thread.AbstractWaiter
import com.vortexdbg.thread.Waiter

abstract class AndroidWaiter : AbstractWaiter(), Waiter {

    override fun onContinueRun(emulator: Emulator<*>) {
    }
}
